packages/gatsby-plugin-sharp/src/report-error.js

```diff
@@ -1,6 +1,10 @@
 const reportError = (message, err, reporter) => {
   if (reporter) {
-    reporter.error(message, err)
+    reporter.error({
+      id: `gatsby-plugin-sharp-20000`,
```

Comment:
without this, messages show up in the console looking like: `ERROR [object Object] ...`

Reply:
it seems like `reporter.error` broke the old-skool way of sending errors?